modules/navier_stokes/src/functormaterials/RhoFromPTFunctorMaterial.C

```diff
@@ -28,7 +28,10 @@ RhoFromPTFunctorMaterial::validParams()
   params.addRequiredParam<MooseFunctorName>(NS::pressure, "pressure functor");
   params.addParam<MooseFunctorName>(
       "density_name", NS::density, "name to use to declare the density functor");
-
+  params.addParam<bool>("neglect_derivatives_of_density_time_derivative",
+                        false,
+                        "Whether to neglect the derivatives with regards to nonlinear variables "
+                        "of the density time derivatives");
```

Comment:
this is for convenience when using this object. not very many properties implement rho(AD, AD, AD &,AD&, AD&)